packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -3,24 +3,30 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import AdmZip from 'adm-zip'
+// import AdmZip from 'adm-zip'
 import os from 'os'
 import fs from 'fs'
 import { parsePatch, applyPatches, ParsedDiff } from 'diff'
 import path from 'path'
 import vscode from 'vscode'
-import { ExportIntent } from '@amzn/codewhisperer-streaming'
+// import { ExportIntent } from '@amzn/codewhisperer-streaming'
 import { TransformByQReviewStatus, transformByQState } from '../../models/model'
-import { ExportResultArchiveStructure, downloadExportResultArchive } from '../../../shared/utilities/download'
+import { ExportResultArchiveStructure } from '../../../shared/utilities/download'
 import { getLogger } from '../../../shared/logger'
 import { telemetry } from '../../../shared/telemetry/telemetry'
 import { CodeTransformTelemetryState } from '../../../amazonqGumby/telemetry/codeTransformTelemetryState'
 import { MetadataResult } from '../../../shared/telemetry/telemetryClient'
 import * as CodeWhispererConstants from '../../models/constants'
-import { createCodeWhispererChatStreamingClient } from '../../../shared/clients/codewhispererChatClient'
+// import { createCodeWhispererChatStreamingClient } from '../../../shared/clients/codewhispererChatClient'
 import { ChatSessionManager } from '../../../amazonqGumby/chat/storages/chatSession'
 import { setContext } from '../../../shared/vscode/setContext'
 
+type PatchDescription = {
```

Comment:
We can just define this once in https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/codewhisperer/models/model.ts

Reply:
Also maybe rename to `PatchInfo` to be more generic